velox/dwio/parquet/writer/arrow/tests/MetadataTest.cpp

```diff
@@ -503,10 +506,12 @@ TEST(Metadata, TestSortingColumns) {
   }
 
   auto sink = CreateOutputStream();
-  auto writerProps = WriterProperties::Builder()
-                         .disable_dictionary()
-                         ->set_sorting_columns(sortingColumns)
-                         ->build();
+  auto writerProps =
+      WriterProperties::Builder()
+          .disable_dictionary()
+          ->set_sorting_columns(sortingColumns)
+          ->created_by(CREATED_BY_VERSION + std::string(" version 1.0"))
```

Comment:
nit: create a variable to ensure an exact match at the input and output validation.
`auto createdBy = CREATED_BY_VERSION + std::string(" version 1.0")`

Reply:
Please rebase when you resolve this.